models/app.model.js

```diff
@@ -20,4 +20,11 @@ exports.checkEndpoints = () => {
       .catch(error => {
         console.error(error);
       });
+}
+
+exports.getArticleById = (id) => {
```

Comment:
your model function should have a name that matches what you're doing here for example selectArticleById or fetchArticleById.